plugins/msr/fu-plugin-msr.c

```diff
@@ -273,7 +276,7 @@ fu_plugin_msr_backend_device_added(FuPlugin *plugin, FuDevice *device, GError **
 		fu_dump_raw(G_LOG_DOMAIN, "IA32_BIOS_SIGN_ID", buf, sizeof(buf));
 		if (!fu_memread_uint32_safe(buf,
 					    sizeof(buf),
-					    0x4,
+					    0x0,
```

Comment:
ya, i need 0x0 on INTEL...

Reply:
On AMD is IA32_BIOS_SIGN_ID even valid?

Reply:
Hmm.  It's the same MSR for both AMD and Intel (0x8b).
* AMD code in the kernel: https://github.com/torvalds/linux/blob/master/arch/x86/kernel/cpu/amd.c#L610
* Intel code in the kernel: https://github.com/torvalds/linux/blob/master/arch/x86/include/asm/microcode_intel.h#L56.

But yeah it looks like in the kernel it's off by 4 for Intel and AMD.  I'll push a fixup.